Integration details: add description of available links (#34771)
home-assistant/home-assistant.github.io: source/getting-started/integration.markdown

   - This opens the integration details page.
   - On the details page, we see that this integration has one {% term service %} and one {% term entity %}.
   - **Documentation** opens the documentation page of that integration.
   - **Known issues** opens the GitHub issue tracker, filtered for issues related to that integrations.

>      - This allows you to see if there is currently an open issue for that integration.

Reviewer: ```suggestion
     - This allows you to see if other users reported issues related to that integration.
```
